src/common/options.cc

```diff
@@ -752,47 +752,47 @@ std::vector<Option> get_global_options() {
     .set_default(false)
     .set_description(""),
 
-    Option("xio_queue_depth", Option::TYPE_INT, Option::LEVEL_ADVANCED)
+    Option("xio_queue_depth", Option::TYPE_SIZE, Option::LEVEL_ADVANCED)
```

Comment:
i agree with @jcsp TYPE_SIZE if for quantities like packet sizes but not for not for settings for numbers, like numbers of packets per seconds.